Galgenmännchen.py

```diff
@@ -0,0 +1,77 @@
+from hangmanStages import stages
+from Words import word_list
+import random
+
+
+def PrintHangmanStage():
+    print(stages[6])
+
+
+def GetRandomWordfromList():
```

Comment:
better return the word instead of creating a global variable here.
The method is called "get..." but doesn't return anything, this should make you say "Something is wrong here" ;)